forge/test/operators/matmul/test_matmul_fuse_bias.py

```diff
@@ -0,0 +1,34 @@
+import pytest
```

Comment:
General comment on import organization (sorted by hierarchy):
1. Native python libraries (e.g. os, math, etc.)
2. Third-party libraries (e.g. torch, numpy, etc.)
3. Our libraries/files (e.g. forge)

Grouping them like this will make our python files a bit more easier to read and follow :))

Reply:
fixed